.github/ISSUE_TEMPLATE/3-docs-bug.md

```diff
@@ -0,0 +1,20 @@
+---
+name: "📚 Docs issue"
+about: Report an issue in webhint's documentation
+
+---
+<!--
+
+Hi there 👋
+
+Before opening a new issue please make sure to search in the existing
```

Comment:
A positive, thankful opening here would be nice (similar to other suggestions above)